src/com/jwetherell/algorithms/mathematics/Division.java

```diff
@@ -3,11 +3,20 @@
 public class Division {
 
     public static final long division(int a, int b) {
+        // Error if divide by zero
+        if (b == 0) {
+            throw new IllegalArgumentException("/ by zero");
+        }
         long result = ((long) a) / ((long) b);
         return result;
     }
 
     public static final long divisionUsingLoop(int a, int b) {
+        // Error if divide by zero
+        if (b == 0) {
+            throw new IllegalArgumentException("/ by zero");
```

Comment:
This will check if the divisor is zero in this method. If this is true, a IllegalArgumentException will be thrown with the error message "/ by zero". This will help in identifying bugs when the divisor is set to 0 unintentionally. It also will prevent unexpected crashes or behaviour.